doc/tech/approach.md

```diff
@@ -3,97 +3,226 @@
+
+For Cloud implementations of multiple well separated simultaneous SW Virtualisation domains on a shared HW Infrastructure there must be a separation of the hardware resources e.g. servers and the Underlay Networking resources that interconnect the hardware resources e.g. through a switching fabric.
+
+To allow multiple separated simultaneous SW Virtualisation domains onto a shared switching fabric there is a need to split up the Underlay Networking resources into non overlapping addressing domains on suitable protocols e.g. VxLAN with their VNI Ranges. This separation must be done through an administrative domain that could not be compromised by any of the individual SW Virtualisation domains either by malicious or unintentional Underlay Network mapping or configuration.
+
+These concepts are very similar to how the Hyperscaler Cloud Providers (HCP) offer Virtual Private Clouds for users of Bare Metal deployment on the HCP shared pool of servers, storage and network resources.
+
+The separation of Hardware and Software Infrastructure Layers makes it important that CNTT Reference Architectures do not include direct management or dependencies of the physical hardware resources e.g. servers and switches inside the HW Infrastructure Layer. All automated interaction from the SW Infrastructure Layer implementations towards the HW and shared networking resources in the HW Infrastructure Layer should go through a common abstracted Reference Model  interface. 
+
+<a name="5.1.3.3"></a>
+#### 5.1.3.3 Software Defined Networking, SDNu and SDNo concepts
+
+A major point with a Cloud Infrastructures is to automate as much as possible. An important tool for Networking automation is Software Defined Networking (SDN) that comes in many different shapes and can act on multiple layers of the networking. In this section we will deal with the internal networking of a data center and not how data centers interconnect with each other or  get access to the world outside of a data center.
+
+When there are multiple simultaneous SW Virtualisation Layers on the same HW Infrastructure, there is a need to ensure Underlay networking separation in the HW Infrastructure Layer. This separation can be done manually through provisioning of a statically configured separation of the Underlay Networking in the HW Infrastructure Layer. A better and more agile usage of the HW Infrastructure is to offer each instance of the SW Virtualisation Layer an SDN interface to the HW Infrastructure. Since these SDN instances only deals with a well separated portion (or slice) of the Underlay Networking we call this interface SDN-Underlay (SDNu).
```

Comment:
"When there are multiple simultaneous SW Virtualisation _Layers_ on the same HW Infrastructure..." - Do we mean multiple layers, or multiple tenants? When I think of multiple layers, I think of them stacked vertically. When I think of tenants, I think of them as being lateral to each other.

Reply:
I generally refrain from using the word tenants when referring to something that could be named “HW Infrastructure Tenants” i.e. a Tenant on top of the HW Infrastructure Layer. The main reason for that is that people often shorten their language and just talk about Tenants and that creates a lot of misunderstandings since the receiver then thinks it is the SW Virtualization Tenants we talk about.

Reply:
Looks good to me in this new version.